.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
   },
   "[json]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "deno.enable": false
```

Comment:
not related?

Reply:
this is to prevent deno plugin in vscode keep prompting error for no `.ts` extension  in import urls
[https://github.com/denoland/vscode_deno](url)

Reply:
It would introduce unnecessary impact of the runtime performance just for very rare user cases., even though it's technically correct.
Just like concept vs real world.
IMO the best way to deal with it is to put this into limitations section.